src/main/java/pl/anasz/cinema/services/RoomsService.java

```diff
@@ -0,0 +1,11 @@
+package pl.anasz.cinema.services;
+
+import pl.anasz.cinema.enities.Room;
+
+import java.util.Optional;
+
+public interface RoomsService {
```

Comment:
Po co stworzyłaś ten (i nastepne) interface'y do serwisów? Jaki jest zysk z tego rozwiązania?

Reply:
Przede wszystkim jest to dobra praktyka, gdyż serwisy należy definiować przez interface'y. Jest to też zachowanie bardziej na przyszłość, ponieważ w przypadku rozrostu aplikacji, jeśli więcej klas implementowałoby ten interface, automatycznie musiałyby implementować też jego metody bez gubienia się w tym gdzie może to być jeszcze wykorzystane. Dodatkowo, jeśli interface ten byłby implementowany w różnych klasach to jego implementacja w poszczególnych klasach mogłaby się różnić w zależności od wymagań biznesowych.

Reply:
Zgadzam się z powyższym. Fajnie że masz świadomość, po co to się tworzy. Warte zastanowienia jest, czy zawsze warto tworzyć tą warstwę, jesli implementuje to tylko jedna klasa. Można to zrobić dopiero w momencie rozszerzania funkcjonalności. Kwestia gustu :).